### PR TITLE
Adding Google-calendar integration for RC

### PR DESCRIPTION
**Pull request by Rohlik**

### CODE REVIEW

File: 5. Administrator Guides/Integrations/Google-Calendar.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Google Calendar
+
+[Google Calendar](https://calendar.google.com/) is calendar from Google company. You can easily organize your daily tasks or meetings and share with your co-workers, etc..
+This integration uses Rocket.chat incoming webhook.
```
Comment (frdmn) [inline 5. Administrator Guides/Integrations/Google-Calendar.md]: To keep our official brand/CI, I would capitalize the "C" of `chat` in this case: Rocket.Chat

File: 5. Administrator Guides/Integrations/Google-Calendar.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Google Calendar
+
+[Google Calendar](https://calendar.google.com/) is calendar from Google company. You can easily organize your daily tasks or meetings and share with your co-workers, etc..
```
Comment (geekgonecrazy) [inline 5. Administrator Guides/Integrations/Google-Calendar.md]: Can you remove "company" from "Google company" ?

**Comment by geekgonecrazy** [comment]
@Rohlik awesome integration!

**Comment by Rohlik** [comment]
@geekgonecrazy and @frdmn Done

**Review by geekgonecrazy — APPROVED** [review]

**Comment by geekgonecrazy** [comment]
@Rohlik thanks!

**Pull request merged**